regparser/web/jobs/models.py

```diff
@@ -1,16 +1,24 @@
 from django.db import models
 
+job_status_pairs = (
+    ("complete", "complete"),
+    ("complete_with_errors", "complete_with_errors"),
+    ("failed", "failed"),
+    ("in_progress", "in_progress"),
+    ("received", "received")
+)
+job_status_values = [j[0] for j in job_status_pairs]
```

Comment:
It should be a tuple; also I meant to use `job_status_pairs` in the arguments for one of the `CharField`s in that file. I've added those changes and will push them shortly.